diff --git a/third_party/nccl/build_defs.bzl.tpl b/third_party/nccl/build_defs.bzl.tpl
index ceb65e3b632f3..c875717403857 100644
--- a/third_party/nccl/build_defs.bzl.tpl
+++ b/third_party/nccl/build_defs.bzl.tpl
@@ -204,7 +204,7 @@ def _prune_relocatable_code_impl(ctx):
             arguments = arguments,
             mnemonic = "nvprune",
         )
-        output.append(outputs)
+        outputs.append(output)
 
     return DefaultInfo(files = depset(outputs))
 
